pom.xml

```diff
@@ -12,449 +12,370 @@
         </license>
     </licenses>
 
-    <properties>
-        <module_name>mod-invoice-storage</module_name>
-        <http.port>8081</http.port>
-        <ramlfiles_path>${basedir}/ramls</ramlfiles_path>
-        <ramlfiles_util_path>${basedir}/raml-util</ramlfiles_util_path>
-        <raml-module-builder.version>23.3.0</raml-module-builder.version>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <java.version>1.8</java.version>
-        <!-- Dependencies versions-->
-        <aspectj.version>1.8.9</aspectj.version>
-        <postgres.driver.version>9.4.1212</postgres.driver.version>
-        <rest.assured.version>3.1.1</rest.assured.version>
-        <junit.version>4.12</junit.version>
-        <apache.http.client.version>4.5.3</apache.http.client.version>
-        <vertx.stack.depchain.version>3.5.4</vertx.stack.depchain.version>
-        <jackson.version>2.9.3</jackson.version>
-        <!-- Plugins versions -->
-        <maven.compiler.plugin.version>3.1</maven.compiler.plugin.version>
-        <build.helper.maven.plugin.version>3.0.0</build.helper.maven.plugin.version>
-        <exec.maven.plugin.version>1.5.0</exec.maven.plugin.version>
-        <aspectj.maven.plugin.version>1.8</aspectj.maven.plugin.version>
-        <maven.resources.plugin.version>3.0.1</maven.resources.plugin.version>
-        <copy.rename.maven.plugin.version>1.0</copy.rename.maven.plugin.version>
-        <maven.antrun.plugin.version>1.8</maven.antrun.plugin.version>
-        <properties.maven.plugin.version>1.0.0</properties.maven.plugin.version>
-        <maven.shade.plugin.version>2.3</maven.shade.plugin.version>
-        <maven.release.plugin.version>2.5.3</maven.release.plugin.version>
-        <maven.surefire.plugin>2.22.1</maven.surefire.plugin>
-    </properties>
+  <properties>
+    <raml-module-builder.version>23.3.0</raml-module-builder.version>
```

Comment:
Please use the most recent [RMB release](https://github.com/folio-org/raml-module-builder/releases) version (a the moment it is 23.9.0)

Reply:
Done